flexget/components/imdb/api.py

```diff
@@ -1,13 +1,15 @@
+from __future__ import absolute_import, division, unicode_literals
```

Comment:
something go wrong and this file got reverted to an old version? Seems like utils/imdb.py showed back up again too